packages/api/src/period/controllers.ts

```diff
@@ -205,9 +188,28 @@ const assignQuantifiersDryRun = async (
         },
       },
     },
+
+    // Sort decsending as first step of "First Fit Decreasing" bin-packing algorithm
+    {
+      $sort: {
+        praiseCount: -1
+      }
+    }
   ]);
 
-  const quantifierPool = await UserModel.aggregate([
+  const totalReceiverPraise = sum(receivers.map((r) => r.praiseIds.length));
+
+  // Run "First Fit" bin-packing algorithm on list of receivers
+  //    with a maximum 'bin' size of: PRAISE_PER_QUANTIFIER * tolerance
+  //    where each item takes up bin space based on its' praiseCount
+  const result: PackingOutput<Receiver> = firstFit(receivers, (r: Receiver) => r.praiseCount, maxPraisePerQuantifier);
+  const bins: Receiver[][] = [...result.bins, ...result.oversized.map((r) => [r])];
+
+  // Clone the bins for each redundant assignment (as defined by setting PRAISE_QUANTIFIERS_PER_PRAISE_RECEIVER)
+  const redundantAssignmentBins: Receiver[][] = flatten(range(quantifiersPerPraiseReceiver).map((i) => bins.slice()))
```

Comment:
Is this intentional?

![image](https://user-images.githubusercontent.com/9698363/156720306-555e3f5e-0c2a-4ccf-8673-8addb7aaa1e1.png)

Reply:
oops - i think my eslint-vscode extension wasn't running. Fixed.